skimage/util/crop.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+
+__all__ = ['crop']
+
+
+def crop(image, bounding_box, axis=None):
+    """Cropping images from a bounding box.
+        Bounding_box (which is a 2-tuple (min_val, max_val) for each axis)
+        and (optional) axis for corresponding axis order to bounding_box.
+
+    Parameters
+    ----------
+    Image : ndarray
+        Input array.
+    Bounding_box : list of 2-tuple (x, y) where x < y.
+        Bounding box.
+    axis : tuple, optional
+        Axis order for cropping.
+        if provided, same legth as bounding_box.
+        Default: None
```

Comment:
```suggestion
        If provided, same length as bounding_box.
```
@rfezzani we don't mention default values in parameter descriptions, right?

Reply:
Yes, absolutely @mkcor. The rational is that it is a repetition between function's signature and its documentation:wink:.

Reply:
@rfezzani makes sense, thanks for confirming :slightly_smiling_face: